recaptcha_enterprise/demosite/src/main/resources/static/scripts/global.js

```diff
@@ -12,48 +12,126 @@
-  // SEE: If using a library or framework, can fetch its usual way
-  return fetchDemoAssessment({ action, token });
+async function homepage({token}) {
+  const json_body = JSON.stringify({
+    recaptcha_cred: {
+      token: token,
+    },
+  });
+  // Fetch the demo assessment from backend.
+  const score = await fetchDemoAssessment({
+    url: SUBMIT_URLS.home,
+    json_body: json_body,
+  });
+  useAssessment(score);
 }
 
-function useAssessment(score) {
-  // SEE: Code for handling the assessment goes here
-  showAssessmentInDemo(score);
+async function signup({token}) {
+  // Do not pass plain text credentials. Always encrypt and follow security standards.
+  // Obtain the values from your form data.
+  const json_body = JSON.stringify({
+    recaptcha_cred: {
+      token: token,
+      username: "",
+      password: "",
+    },
+  });
+  // Fetch the demo assessment from backend.
+  const score = await fetchDemoAssessment({
+    url: SUBMIT_URLS.signup,
+    json_body: json_body,
+  });
+  useAssessment(score);
 }
 
-// ATTENTION: reCAPTCHA Example (Part 3) Ends
+async function login({token}) {
+  // Do not pass plain text credentials. Always encrypt and follow security standards.
+  // Obtain the values from your form data.
+  const json_body = JSON.stringify({
+    recaptcha_cred: {
+      token: token,
+      username: "",
+      password: "",
+    },
+  });
+  // Fetch the demo assessment from backend.
+  const score = await fetchDemoAssessment({
+    url: SUBMIT_URLS.login,
+    json_body: json_body,
+  });
+  useAssessment(score);
+}
 
-function fetchDemoAssessment({ action, token }) {
-  const body = JSON.stringify({
+async function store({token}) {
+  // Do not pass plain text credentials. Always encrypt and follow security standards.
+  // Obtain the values from your form data.
+  const json_body = JSON.stringify({
     recaptcha_cred: {
-      action,
-      token,
+      token: token,
+      items: {}.toString(),
     },
   });
-  return fetch("/create_assessment", {
-    body,
+  // Fetch the demo assessment from backend.
+  const score = await fetchDemoAssessment({
+    url: SUBMIT_URLS.store,
+    json_body: json_body,
+  });
+  useAssessment(score);
+}
+
+async function comment({token}) {
+  // Do not pass plain text credentials. Always encrypt and follow security standards.
+  // Obtain the values from your form data.
+  const json_body = JSON.stringify({
+    recaptcha_cred: {
+      token: token,
+      comment: "",
+    },
+  });
+  // Fetch the demo assessment from backend.
+  const score = await fetchDemoAssessment({
+    url: SUBMIT_URLS.comment,
+    json_body: json_body,
+  });
+  useAssessment(score);
+}
+
+function fetchDemoAssessment({ url, json_body}) {
+  // Code for fetching the assessment from backend goes here.
+  // Refer to demo app backend code for more information.
+  // See if using a library or framework, can use event handlers its usual way.
```

Comment:
This line is unclear. Can we try to rephrase it?

Reply:
Update for more clarity.